HunterPie.Core/Core/Local/Player.cs

```diff
@@ -830,6 +849,17 @@ private void GetPlayerEffects()
             HasHotDrink = hotDrinkFlag;
         }
 
+        private void GetHUDState()
+        {
+            // Could read in other HUD details if that matters to anyone
+
+            var hasHUDActiveAddress = Kernel.ReadMultilevelPtr(
+                Address.GetAddress("BASE") + Address.GetAddress("GAME_HUD_INFO_OFFSET"),
+                new[] { 0x14300, 0x2900 });
```

Comment:
Could add those offsets to the map too instead of keeping them hardcoded, but it's fine I think since the game won't really be updated anymore I think?